<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(30, 30, 30)</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>50</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 87 46pt &quot;Arial Black&quot;;
</string>
   </property>
   <property name="text">
    <string>REGISTER</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="username">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>200</y>
     <width>441</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	border: 2px solid rgb(42, 42, 42);
	border-radius: 20px;
	color: #FFF;
	padding-left:20px;
	padding-right:20px;
	background-color:rgb(45, 45, 45);
	font-size: 28px;
}</string>
   </property>
   <property name="placeholderText">
    <string>Username</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="password">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>280</y>
     <width>441</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	border: 2px solid rgb(42, 42, 42);
	border-radius: 20px;
	color: #FFF;
	padding-left:20px;
	padding-right:20px;
	background-color:rgb(45, 45, 45);
	font-size: 28px;
}</string>
   </property>
   <property name="placeholderText">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="registerButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>500</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 255, 127);
font: 87 18pt &quot;Arial Black&quot;;
border-radius: 18px;</string>
   </property>
   <property name="text">
    <string>REGISTER</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="confirmPassword">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>360</y>
     <width>441</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	border: 2px solid rgb(42, 42, 42);
	border-radius: 20px;
	color: #FFF;
	padding-left:20px;
	padding-right:20px;
	background-color:rgb(45, 45, 45);
	font-size: 28px;
}</string>
   </property>
   <property name="placeholderText">
    <string>Confirm password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>550</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 255, 127);
font: 87 18pt &quot;Arial Black&quot;;
border-radius: 18px;</string>
   </property>
   <property name="text">
    <string>BACK</string>
   </property>
  </widget>
  <widget class="QLabel" name="register_error_label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>450</y>
     <width>461</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 12pt &quot;Arial Black&quot;;
color: red;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>660</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 87 10pt &quot;Arial Black&quot;;
</string>
   </property>
   <property name="text">
    <string>Â© Created by Krylov</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="Resource/login_form.qrc"/>
 </resources>
 <connections/>
</ui>
